.github/workflows/gradle.yml

```diff
@@ -0,0 +1,22 @@
+name: Gill Bates
```

Comment:
:thinking: 
Maybe something like `Gradlew build`, `Build`, `KAMI`? Not.. Gill Bates.

Reply:
Mine is just Java CI which is the default name, and makes enough sense because it *is* just a Java Continuous Integration service

Reply:
:trollface:  
Okay gonna change to `Java CI`

Reply:
fixed [here](https://github.com/zeroeightysix/KAMI/pull/136/commits/f479685f5cc61b41678aa499bf0892f4b638d090)